 swat.c - patch is currently disabled because tme man-section 5 is not converted to 4
# git-like
#--- a/source3/web/swat.c	2009-07-03 04:21:14.000000000 -0700
#+++ b/source3/web/swat.c	2009-09-01 07:39:04.758980165 -0700
#@@ -203,12 +203,12 @@
# 	char *output;
# 	if(strcmp(pLabel, pTranslated) != 0) {
# 		output = talloc_asprintf(ctx,
#-		  "<A HREF=\"/swat/help/manpages/smb.conf.5.html#%s\" target=\"docs\"> %s</A>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; %s <br><span class=\"i18n_translated_parm\">%s</span>",
#+		  "<A HREF=\"/swat/help/manpages/smb.conf.4.html#%s\" target=\"docs\"> %s</A>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; %s <br><span class=\"i18n_translated_parm\">%s</span>",
# 		   pAnchor, pHelp, pLabel, pTranslated);
# 		return output;
# 	}
# 	output = talloc_asprintf(ctx,
#-	  "<A HREF=\"/swat/help/manpages/smb.conf.5.html#%s\" target=\"docs\"> %s</A>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; %s",
#+	  "<A HREF=\"/swat/help/manpages/smb.conf.4.html#%s\" target=\"docs\"> %s</A>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; %s",
# 	  pAnchor, pHelp, pLabel);
# 	return output;
# }
#@@ -1257,7 +1257,7 @@
#         printf("<H3>%s</H3>\n", _("Important Note:"));
#         printf("%s",_("Printer names marked with [*] in the Choose Printer drop-down box "));
#         printf("%s",_("are autoloaded printers from "));
#-        printf("<A HREF=\"/swat/help/smb.conf.5.html#printcapname\" target=\"docs\">%s</A>\n", _("Printcap Name"));
#+        printf("<A HREF=\"/swat/help/smb.conf.4.html#printcapname\" target=\"docs\">%s</A>\n", _("Printcap Name"));
#         printf("%s\n", _("Attempting to delete these printers from SWAT will have no effect."));
# 
# 	if (cgi_variable("Commit") && snum >= 0) {
# git-like
--- a/docs-xml/configure.ac-bkp	2011-05-23 11:40:30.309466703 +0200
+++ b/docs-xml/configure.ac	2011-05-23 12:18:34.727978926 +0200
@@ -54,5 +54,8 @@
 fi
 
 AC_SUBST(TARGETS)
+AC_SUBST(BUILDDIR)
+AC_SUBST(DOCBOOKXSLVER)
+AC_SUBST(DOCBOOKXSLDIR)
 
 AC_OUTPUT( Makefile.settings build/catalog.xml)
# git-like
--- a/docs-xml/smbdotconf/generate-file-list.sh	Sat Feb 20 01:12:25 2010
+++ b/docs-xml/smbdotconf/generate-file-list.sh	Mon Jan 18 03:38:09 2010
@@ -9,7 +9,10 @@
 cd $DIR
 
 echo "<section xmlns:xi=\"http://www.w3.org/2003/XInclude\">"
-for I in `find . -mindepth 2 -type f -name '*.xml' | sort -t/ -k3 | xargs`
+
+#for I in `find . -mindepth 2 -type f -name '*.xml' | sort -t/ -k3 | xargs`
+# Remove lines containing just one slash
+for I in `find . -type f -name '*.xml' | grep -v '^[^/]*/[^/]*$' | sort -t/ -k3 | xargs`
 do 
 	echo "<xi:include href='$I' parse='xml'/>"
 done
# git-like
--- a/docs-xml/build/catalog.xml.in-bkp	2011-05-23 11:41:02.207116056 +0200
+++ b/docs-xml/build/catalog.xml.in	2011-05-23 12:09:22.968664219 +0200
@@ -2,6 +2,49 @@
 <!DOCTYPE catalog PUBLIC "-//OASIS//DTD Entity Resolution XML Catalog V1.0//EN" "http://www.oasis-open.org/committees/entity/release/1.0/catalog.dtd">
 <catalog xmlns="urn:oasis:names:tc:entity:xmlns:xml:catalog">
 
+<!-- There is obsolete release of the docbook-xml in Solaris so we should
+     download more recent release and rewrite references to it -->
+
+   <delegatePublic
+	publicIdStartString="-//OASIS//ENTITIES DocBook XML"
+	catalog="file:///usr/share/sgml/docbook/xmlcatalog"/>
+
+   <delegatePublic
+	publicIdStartString="-//OASIS//DTD DocBook XML"
+	catalog="file:///usr/share/sgml/docbook/xmlcatalog"/>
+
+   <delegatePublic
+	publicIdStartString="ISO 8879:1986"
+	catalog="file:///usr/share/sgml/docbook/xmlcatalog"/>
+
+   <delegateSystem
+	systemIdStartString="http://www.oasis-open.org/docbook/"
+	catalog="file:///usr/share/sgml/docbook/xmlcatalog"/>
+
+   <delegateURI
+	uriStartString="http://www.oasis-open.org/docbook/"
+	catalog="file:///usr/share/sgml/docbook/xmlcatalog"/>
+
+   <rewriteSystem
+	systemIdStartString="http://docbook.sourceforge.net/release/xsl/@DOCBOOKXSLVER@"
+	rewritePrefix="file:///@DOCBOOKXSLDIR@"/>
+
+   <rewriteURI
+	uriStartString="http://docbook.sourceforge.net/release/xsl/@DOCBOOKXSLVER@"
+	rewritePrefix="file:///@DOCBOOKXSLDIR@"/>
+
+   <rewriteSystem
+	systemIdStartString="http://docbook.sourceforge.net/release/xsl/current"
+	rewritePrefix="file:///@DOCBOOKXSLDIR@"/>
+
+   <rewriteURI
+	uriStartString="http://docbook.sourceforge.net/release/xsl/current"
+	rewritePrefix="file:///@DOCBOOKXSLDIR@"/>
+
+   <rewriteSystem
+	systemIdStartString="http://scrollkeeper.sourceforge.net/dtds/scrollkeeper-omf-1.0"
+	rewritePrefix="file:///usr/share/xml/scrollkeeper/dtds"/>
+
 <!--  XML Catalog file for SAMBA. -->
 
     <rewriteURI
